enterprise/maven.j2ee/nbproject/project.properties

```diff
@@ -16,7 +16,7 @@
 # under the License.
 
 is.eager=true
-javac.compilerargs=-Xlint -Xlint:-serial -Xlint -Xlint:-serial
+javac.compilerargs=-Xlint -Xlint:-serial
 
 javac.source=1.8
```

Comment:
i think CI should pass once the other two modules are also bumped to `release=17`